conf/machine/hikey-32.conf

```diff
@@ -0,0 +1,45 @@
+#@TYPE: Machine
+#@NAME: LeMaker HiKey machine
+#@DESCRIPTION: LeMaker HiKey machine
+
+require conf/machine/include/tune-cortexa8.inc
+DEFAULTTUNE = "cortexa8hf-neon"
```

Comment:
Drop DEFAULTTUNE, the default from tune-cortexa8.inc works perfectly well and doesn't break package feeds and sstate like the cortex* tunes do.